core/sql/sqlcomp/CmpSeabaseDDLroutine.cpp

```diff
@@ -348,20 +348,32 @@ void CmpSeabaseDDL::createSeabaseLibrary(
       processReturn();
       return;
     }
+
+  ComTdbVirtTableTableInfo * tableInfo = new(STMTHEAP) ComTdbVirtTableTableInfo[1];
```

Comment:
these assignments should be part of the arguments to a constructor to ComTdbVirtTableTableInfo.